packages/components/src/Chips/useAssert.tsx

```diff
@@ -0,0 +1,15 @@
+interface Options {
+  readonly warn: boolean;
+}
+
+export function useAssert(
```

Comment:
This seems pretty generic, should this sit outside Chips? Is this reusable for other components?

Reply:
Was planning to do that in a different PR :P